omnibus/files/private-chef-cookbooks/private-chef/attributes/default.rb

```diff
@@ -476,7 +476,7 @@
 # - AES added for pushy-server 1.x compatibility
 # - TLS1 and TLSv1.1 added for pushy-server 1.x compatibility
 #
-default['private_chef']['nginx']['ssl_protocols'] = 'TLSv1 TLSv1.1 TLSv1.2'
+default['private_chef']['nginx']['ssl_protocols'] = 'TLSv1.2'
```

Comment:
We should document the versions of chef-client that we will be incompatible with after this update.

Reply:
I have the versions ready for the release notes and I'll make sure we get that for the docs. The docs right now actually make it sounds like we support SSL 3.0 by default. They need some work.